### PR TITLE
chore: set up travis

### PR DESCRIPTION
**Pull request by SteveRedka**
## Checklist

- [x] Tests (if applicable)
- [x] Documentation (if applicable)
- [ ] Changelog stub (or use [conventional commit messages](https://www.conventionalcommits.org/))

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,30 @@
...
+
 language: ruby
-cache: bundler
+
 rvm:
-  - 2.6.3
-before_install: gem install bundler -v 1.17.2
+  - 2.5
+  - 2.6
+  - 2.7
```
Comment (dmitry-mukhin) [inline .travis.yml]: will this run on older rubies?

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,30 @@
...
+
+before_install:
+  - gem uninstall bundler
+  - gem install bundler --pre
+  - bundle update --bundler
+
+jobs:
+  include:
+    - stage: style checks
+      rvm: 2.4
```
Comment (dmitry-mukhin) [inline .travis.yml]: should we run style check on the most recent ruby?

File: .travis.yml [hunk 3]
```diff
@@ -1,7 +1,30 @@
...
+      before_install:
+        - gem install rubocop
+      script:
+        - rubocop
+
+  allow_failures:
+    - stage: style checks
+    - rvm: 2.6
```
Comment (dmitry-mukhin) [inline .travis.yml]: this doesn't make sense, i think as style checks are not run on 2.6

File: .travis.yml [hunk 4]
```diff
@@ -1,7 +1,30 @@
...
+  - 2.6
+  - 2.7
+
+env:
+  UPLOADCARE_PUBLIC_KEY=demopublickey
+  UPLOADCARE_SECRET_KEY=demoprivatekey
+
+before_install:
+  - gem uninstall bundler
+  - gem install bundler --pre
```
Comment (dmitry-mukhin) [inline .travis.yml]: why do we need this?

File: .travis.yml [hunk 5]
```diff
@@ -1,7 +1,30 @@
----
-sudo: false
+# currently supported rubies: https://www.ruby-lang.org/en/downloads/branches/
+
 language: ruby
-cache: bundler
```
Comment (dmitry-mukhin) [inline .travis.yml]: and why don't we need this?

File: .travis.yml (hunk 1 above)
Comment (SteveRedka) [reply .travis.yml]: dry-configurable requires 2.4, so no. Should I look into this?

Comment (dmitry-mukhin) [reply .travis.yml]: nope its fine, I just want to know why we can't run on 1.9

**Pull request merged**